gc/base/GCExtensionsBase.hpp

```diff
@@ -1071,6 +1075,9 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 		, scvTenureStrategyLookback(true)
 		, scvTenureStrategyHistory(true)
 		, scavengerEnabled(false)
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+		, concurrentScavenger(true)
```

Comment:
If the define is on we should be enabling the concurrentScavenger by default?  I think it is ok since no one will be enabling this feature for now but something to keep track of